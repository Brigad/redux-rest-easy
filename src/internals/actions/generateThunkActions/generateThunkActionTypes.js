import generateActionType from '../../utils/generateActionType';

const generateThunkActionTypes = resourceName => ({
  INVALIDATE_REQUEST: generateActionType(
    resourceName,
    '@invalidate',
    'INVALIDATE_REQUEST',
  ),
});

export default generateThunkActionTypes;
