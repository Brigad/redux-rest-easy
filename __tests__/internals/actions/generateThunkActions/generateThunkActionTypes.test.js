import generateThunkActionTypes from '../../../../src/internals/actions/generateThunkActions/generateThunkActionTypes';

describe('generateThunkActionTypes', () => {
  test('only path', () => {
    expect(generateThunkActionTypes('fruits')).toMatchSnapshot();
  });
});
