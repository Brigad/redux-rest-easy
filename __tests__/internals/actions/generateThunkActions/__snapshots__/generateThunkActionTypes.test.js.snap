// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`generateThunkActionTypes only path 1`] = `
Object {
  "INVALIDATE_REQUEST": "@@rest-easy/fruits/@invalidate/INVALIDATE_REQUEST",
}
`;
